lib/tasks/multitenant/triggers.rake

```diff
@@ -502,40 +491,35 @@ namespace :multitenant do
           SET NEW.tenant_id = NEW.provider_account_id;
         END IF;
 
-        SET @numbering_period = (SELECT numbering_period
-                                 FROM billing_strategies
-                                 WHERE account_id = NEW.provider_account_id
-                                 LIMIT 1);
-
-        IF @numbering_period = 'monthly' THEN
-          SET @invoice_prefix_format = "%Y-%m";
-        ELSE
-          SET @invoice_prefix_format = "%Y";
-        END IF;
+        IF NEW.friendly_id IS NOT NULL AND NEW.friendly_id <> 'fix' THEN
```

Comment:
maybe dry up this conditional?

Reply:
Any suggestion? Because I think it's pretty straightforward the way it is now. `NEW.friendly_id` has to be different than `fix` – which is the default value for the column – but also `NOT NULL`. Of course `NULL` is different than `fix`.

Reply:
The (other one)[https://github.com/3scale/porta/blob/33fbd6d8bc2955c53ce52dff8c48d3229ee5128e/lib/tasks/multitenant/triggers.rake#L100] is in the definition of the Oracle trigger. This one is Mysql. In both cases, `NEW` (Mysql) or `:new` (Oracle) are pointers to the current record being inserted.